python/fledge/tasks/north/sending_process.py

```diff
@@ -925,6 +925,9 @@ async def _start(self):
 
             exec_sending_process = self._config['enable']
 
+            if self.isDryRun():
+                return
```

Comment:
With --dryrun Sending process logs are coming with Started....Stop...Execution Completed, which means EGRESS of readings is going on if north server is reachable.